tests/commands/update/test_update_case_cmd.py

```diff
@@ -39,8 +39,8 @@ def test_update_case_existing_collaborator(mock_app, case_obj):
     ## WHEN providing a case id and a valid collaborator
     result = runner.invoke(cli, ["update", "case", case_obj["_id"], "-c", "cust000"])
 
-    ## THEN assert that the case is fetched
-    assert "INFO Fetching case {}".format(case_obj["_id"]) in result.output
```

Comment:
testing for the info output.. Let us just say with confidence that there is more than one way to do it.